cli/azd/internal/appdetect/java.go

```diff
@@ -15,7 +15,8 @@ import (
 )
 
 type javaDetector struct {
-	rootProjects []mavenProject
+	rootProjects      []mavenProject
+	mavenWrapperPaths [][]string
```

Comment:
mavenWrapperPaths[i][0] is used for linux, and mavenWrapperPaths[i][1] is for win, it's error prone to use index to differentiate them. How about using new struct and const to differentiate them?

Reply:
fixed